packages/core/src/drivers/db/models/modalidad.js

```diff
@@ -1,6 +1,6 @@
 const { Model, DataTypes, Sequelize } = require('sequelize');
 
-const MODALIDAD_TABLE = 'Modalidades';
+const MODALIDAD_TABLE = 'modalidades';
```

Comment:
Let's keep this like it was before to have a standard

Reply:
In this case, it was a typo, because all the tables are in lowercase.